assets/navigations/components/NavigationBar.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { gettext } from 'utils';
+import SearchBar from 'wire/components/SearchBar';
+
+class NavigationBar extends React.Component {
```

Comment:
imo there should be a component for all those SomethingBars, atm it's duplicated everywhere with different label and callbacks